src/taxonomy/import-tags/data/api.js

```diff
@@ -40,19 +50,62 @@ export async function importNewTaxonomy(taxonomyName, taxonomyDescription, file)
 }
 
 /**
- * Import tags to an existing taxonomy, overwriting existing tags
+ * Build the mutation to import tags to an existing taxonomy
+ * @returns {import("@tanstack/react-query").UseMutationResult}
+ */
+export const useImportTags = () => {
+  const queryClient = useQueryClient();
+  return useMutation({
+    /**
+    * @type {import("@tanstack/react-query").MutateFunction<
+    *   any,
+    *   any,
+    *   {
+    *     taxonomyId: number
+    *     file: File
+    *   }
+    * >}
+    */
+    mutationFn: async ({ taxonomyId, file }) => {
+      const formData = new FormData();
+      formData.append('file', file);
+
+      await getAuthenticatedHttpClient().put(
+        getTagsImportApiUrl(taxonomyId),
+        formData,
+      );
+    },
+    onSuccess: (_data, variables) => {
+      queryClient.invalidateQueries({
+        queryKey: ['tagList', variables.taxonomyId],
```

Comment:
@rpenido I think also you need to invalidate the `taxonomyDetail` query

Reply:
Thank you @ChrisChV! I missed the tag count on Taxonomy.
Done c6cf04ddc73e7f632d2241dc039ee9c92e9439ca